src/main/java/frc/robot/RobotContainer.java

```diff
@@ -71,16 +71,33 @@ public RobotContainer() {
 
     Shuffleboard.getTab("Subsystems").add(drive.getName(), drive);
     Shuffleboard.getTab("Subsystems").add(intake.getName(), intake);
-    Shuffleboard.getTab("Subsystems").add(intakeLimelight.getName(), intakeLimelight);
+    // Shuffleboard.getTab("Subsystems").add(intakeLimelight.getName(), intakeLimelight);
     Shuffleboard.getTab("Subsystems").add(tagLimelight.getName(), tagLimelight);
-    Shuffleboard.getTab("Subsystems").add(lights.getName(), lights);
+    // Shuffleboard.getTab("Subsystems").add(lights.getName(), lights);
```

Comment:
I understand these being commented out, if we think it's beneficial to sometimes want to view them in shuffleboard but want to avoid the processing/network power of always displaying them. If that isn't the case, they should be uncommented or removed.

Reply:
Yeah we're gonna want these around to comment / uncomment